### ISSUE TITLE
Fail to load bullet-native

### ISSUE THREAD

**Issue by MeFisto94** [jMonkeyEngine/sdk#228]
```
The required native library 'bulletjme' was not found in the classpath via 'native/linux/x86_64/libbulletjme.so'. Error message: no bulletjme in java.library.path: [/usr/java/packages/lib, /usr/lib/x86_64-linux-gnu/jni, /lib/x86_64-linux-gnu, /usr/lib/x86_64-linux-gnu, /usr/lib/jni, /lib, /usr/lib]
```
Not sure what is wrong there again, but I am short of reverting netbeans 11 and stick with java 8 and netbeans 8.2. The engine itself doesn't use anything other than 8 anyway

**Comment by MeFisto94** [comment]
Fixed in https://github.com/jMonkeyEngine/jmonkeyengine/pull/1255

**Comment by MeFisto94** [comment]
So: This was a build error and it has been fixed and merged into `master`, not cherry picked into 3.3, but I think 70626cd22cf976b66dfaec534768882c5d73b855 solved that.
